### PR TITLE
Update known-issues.md

### PR DESCRIPTION
**Pull request by paol**
Add findings from issue #904 to the known issues doc

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
/assign @BenTheElder 

Thanks @paol for your contribution, please check if you can sign the CLA per https://github.com/kubernetes-sigs/kind/pull/917#issuecomment-537945005

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -236,6 +236,22 @@ an 03 17:42:41 kind-1-control-plane kubelet[3804]: F0103 17:42:41.470269 3804 ku
 
 This problem seems to be related to a [bug in Docker][moby#9939].
 
+## Pod errors due to "too many open files"
```
Comment (aojea) [inline site/content/docs/user/known-issues.md]: Please add an entry in the contents section https://github.com/kubernetes-sigs/kind/pull/917/files#diff-a43cc0c4cf108c28fe507969c9eaa365L19

Comment (paol) [reply site/content/docs/user/known-issues.md]: Done

**Comment by aojea** [comment]
SGTM

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Pull request merged**